examples/tests/Earthfile

```diff
@@ -582,8 +582,10 @@ cache-mount-arg:
     RUN cat output.txt | grep '\*cached\* --> RUN echo Doing something 2'; test "$?" != 0
 
 true-false-flag:
+    RUN --no-cache rm -f /var/log/buildkitd.log output.txt
```

Comment:
What's this for?

Reply:
oh no, clearly wasn't supposed to have been added :facepalm: